packages-exp/analytics-compat/package.json

```diff
@@ -56,5 +55,14 @@
       ".ts"
     ],
     "reportDir": "./coverage/node"
-  }
-}
+  },
+  "exports": {
+    "main": "./dist/index.cjs.js",
+    "esm5": "./dist/index.esm.js",
+    "browser": "./dist/index.esm2017.js",
+    "module": "./dist/index.esm2017.js",
+    "node": "./dist/index.cjs.js",
+    "default": "./dist/index.esm2017.js"
+  },
```

Comment:
I'm not sure, webpack seems to look at a lot of possible fields https://webpack.js.org/guides/package-exports and we don't distinguish between browser/module/react-native/etc in the minimal firebase-exp entry points so if you had an app that cared about the difference it would only find them in the individual `@firebase` packages entry points.